### ISSUE TITLE
Redstone logic time

### ISSUE THREAD

**Issue by xpghard** [Imanity-Software/ImanitySpigot3-git#26]
### Stack trace

```
paste your stack trace or a https://paste.imanity.dev/ link here!
```
https://timin.gs/?id=833ddf78f8534e4c82783ad7b6e00f39

### Plugin list

0 plugins

### Steps/models to reproduce (if known)

simply redstone clock https://gyazo.com/3d7a3ef6b15234eff2033d977cd40126

### ImanitySpigot3 version

spigotJarVersion: 2021.08.1 BUILD 7

### Agreements

- [X] I am running the latest version of ImanitySpigot3 available from
- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have compared the issue inbetween TacoSpigot 1.8.8 and ImanitySpigot3 And only ImanitySpigot3 occured this issue.

**Comment by LeeGodSRC** [comment]
**WRONG FORMAT**
You should report this as Bug Report format next time

**Comment by LeeGodSRC** [comment]
Can you provide a spark report?

**Comment by xpghard** [comment]
Yea this is spark report 
https://spark.lucko.me/q4UWBy4tx5

**Comment by LeeGodSRC** [comment]
This issue will be fixed in **2021.08.1 BUILD 8**
